packages/peregrine/lib/talons/CartPage/ProductListing/useProduct.js

```diff
@@ -18,6 +18,7 @@ export const useProduct = props => {
     const [{ cartId }] = useCartContext();
 
     const [isFavorite, setIsFavorite] = useState(false);
+    const [updateItemErrorMessage, setUpdateItemErrorMessage] = useState(null);
```

Comment:
My ignorance with apollo errror handling is showing but could we not just monitor the original mutation state? I don't know if that is cleared between requests but if it is it would prevent the need for this local state.

Something like

```js
const [updateItemQuanitty, { error: updateItemErrorMessage }] = useMutation(updateItemQuantityMutation);
useEffect(() => {
 if (updateItemErrorMessage) { ... }
}, [updateItemErrorMessage]);
```

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2196/commits/47dd63e778e8576be33ef27c626cdf492a022deb.